smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java

```diff
@@ -222,6 +223,20 @@ private void bindCommandsToDriverXboxButtons(){
               "auto balance")
             )
           );
+
+          this.driverController.povRight().whileTrue(
```

Comment:
Seems like there is little protection from accidental deploy of fine placement.  If the robot is at high speed, and then the POV is touched, could the robot will flip?

Reply:
Yes, it would.  Good catch.  I can't think of an easy fix for this.  They only way I can think of to address this is to push the acceleration control (which is currently in the default drive command) down into the drivetrain subsystem.  Then, drivetrain subsystem handles acceleration control for everything that uses it (default drive command, left trigger, trajectories, and this command).  The downside is that we wouldn't be able to customize acceleration control for different modes.  

We can't handle this the same way we handle the left trigger low speed mode (ramping up and ramping down) because John likely will be using both at the same.  This command would grab the DriveTrainPowerSubsystem and nullify the effect of the left trigger even though John has it held down.  It would cause unexpected behavior when he released the dpad while still holding left trigger (the robot would go back to full speed rather than the expected 20% mode).  

Any ideas?

Reply:
I implemented the change to push the acceleration control down into DrivetrainSubsystem.